platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/diagram-commands.txt

```diff
@@ -10,15 +10,16 @@ pcli diagram \
     -s 'eventCreationManager:non-validated events:†' \
     -s 'applicationTransactionPrehandler:futures:★' \
     -s 'pcesReplayer:done streaming pces:@' \
-    -s 'extractOldestMinimumGenerationOnDisk:minimum generation to store:s' \
+    -s 'extractOldestMinimumGenerationOnDisk:minimum identifier to store:s' \
+    -s 'linkedEventIntake:flush request:f' \
     -l 'eventDurabilityNexus:countUpLatch:linkedEventIntake' \
```

Comment:
while you're at it - this manual link is incorrect, and can be removed .

Reply:
removed manual link for event durability nexus

Reply:
With this removed wire, I don't have to substitute a flush request, it's now no longer a cycle.